akka-docs/src/main/paradox/mailboxes.md

```diff
@@ -74,7 +74,7 @@ The type of the queue in the mailbox created for an actor will be checked agains
 ### Requiring a Message Queue Type for a Dispatcher
 
 A dispatcher may also have a requirement for the mailbox type used by the
-actors running on it. An example is the @apidoc[BalancingDispatcher] which requires a
+actors running on it. An example is the `BalancingDispatcher` which requires a
```

Comment:
I removed the link. `BalancingDispatcher` is a private class, it's scaladoc/javadoc is not accessible